### PR TITLE
varnish 4.1.0

### PR DESCRIPTION
**Pull request by denisbr**
Version update to latest available from the Varnish Cache project.

### CODE REVIEW

File: Library/Formula/varnish.rb [hunk 1]
```diff
@@ -1,16 +1,10 @@
 class Varnish < Formula
   desc "High-performance HTTP accelerator"
   homepage "https://www.varnish-cache.org/"
-  url "https://repo.varnish-cache.org/source/varnish-4.0.3.tar.gz"
-  sha256 "94b9a174097f47db2286acd2c35f235e49a2b7a9ddfdbd6eb7aa4da9ae8f8206"
+  url "https://repo.varnish-cache.org/source/varnish-4.1.0.tar.gz"
+  sha256 "4a6ea08e30b62fbf25f884a65f0d8af42e9cc9d25bf70f45ae4417c4f1c99017"
   revision 1
```
Comment (DomT4) [inline Library/Formula/varnish.rb]: Please remove this line on stable version updates, Thanks!

File: Library/Formula/varnish.rb [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
 
-  bottle do
```
Comment (DomT4) [inline Library/Formula/varnish.rb]: You can leave this whole block as-is. We'll update it with the new hashes on pull.

File: Library/Formula/varnish.rb (hunk 1 above)
Comment (denisbr) [inline Library/Formula/varnish.rb]: @DomT4 which line did you want me to remove?

File: Library/Formula/varnish.rb (hunk 2 above)
Comment (denisbr) [inline Library/Formula/varnish.rb]: Ok, I thought the guide said to remove the bottle block, that's why I removed it.

Comment (mikemcquaid) [inline Library/Formula/varnish.rb]: Can you point to what link was confusing so we can make it better? Thanks!

Comment (denisbr) [inline Library/Formula/varnish.rb]: @mikemcquaid When re-reading it now it makes perfect sense, I must have been tired or something.

Comment (mikemcquaid) [inline Library/Formula/varnish.rb]: No worries, thanks!

File: Library/Formula/varnish.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/varnish.rb]: The `revision 1` line.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @denisbr; we appreciate it! :tada:

**Pull request closed without merging**